src-java/kilda-persistence-api/src/main/java/org/openkilda/persistence/tx/TransactionArea.java

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.persistence.tx;
+
+public enum TransactionArea {
```

Comment:
We've already discussed (as a part of #4311) that TransactionArea / TransactionLayout / PersistenceManagerBase.transactionManagerByArea is far from commonly used approaches. It significantly complicates understanding, usage and maintenance of the persistence layer.
The general feeling is over-engineered solution.

Just to remind - I proposed to forbid intersection of orientdb and sql persistence contexts, and force a developer to specify what kind of persistence context is required in each specific case. This solves the problem with less abstraction entities.